applications/tari_dan_wallet_cli/src/main.rs

```diff
@@ -38,21 +38,23 @@ async fn main() -> Result<(), anyhow::Error> {
     log::info!("🌍️ Connecting to {}", endpoint);
     let client = WalletDaemonClient::connect(endpoint)?;
```

Comment:
Perhaps we should set the token in the client to avoid having to pass it around everywhere.

```suggestion
    let mut client = WalletDaemonClient::connect(endpoint)?;
    if let Some(token) = token {
       client.set_jwt(token);
    }
```